source/changelog.md.erb

```diff
@@ -0,0 +1,71 @@
+---
+title: Changelog
+nav_order: 100
+---
+
+### v2.6
+
+We are proud to announce that we have successfully upgraded to Play 2.4.x, "Damiya". Although the last few Play releases were generally straightforward to upgrade to,
+with 2.4, the Play team made a lot of big changes to the framework, resulting in a lot of fixes and performance improvements. Some of the changes that are more relevant
+to Octoparts include:
+
+- Dependency injection out of the box. (Octoparts continues to make use of compile-time DI for correctness and performance)
+- Combination of previously separate thread pools into the Play default thread pool
+- Migration of components out of the Global singleton for easier mocking and better reasonability
+
+After performing the upgrade, we deployed it into our test and QA environments and carefully monitored it for problems. When we found none, we used Siege to stress test the latest Play 2.3.x-based version of Octoparts against the Play 2.4.x-based version.  The following results were obtained from Sieging using a payload equivalent to one that would request data for the M3.com homepage for signed-in users, with 100 concurrent requests, and zero delay on a 2Ghz Corei7 Macbook Pro with 8GB RAM:
+
+#### Play 2.3.x-based Octoparts results
+
+<div class="screenshot screenshot-border">
+
+  <% link_to '/img/opqa1_play23.png', :target => "_blank" do %>
+      <%= image_tag 'opqa1_play23.png', :width => '500px'%>
+      <p>Play 2.3.x-based Octoparts JVisualVM stats</p>
+  <% end %>
+
+</div>
+
+<pre>
```

Comment:
Putting these two side-by-side would make comparisons a bit easier. I think possibly the same for the graphs, but definitely here.